packages/wells/src/client/model/Wellbore.ts

```diff
@@ -15,21 +15,29 @@ export type SearchWellbore = (args?: any) => Promise<Wellbore>;
  */
 export interface Wellbore {
   /**
+   * Wellbore name
+   * @type {string}
+   * @memberof Wellbore
+   */
+  name: string;
+  /**
+   * Asset id in CDF
    * @type {number}
    * @memberof Wellbore
    */
   id: number;
   /**
-   * @type {string}
+   * External Id for the wellbore
+   * @type {string}}
    * @memberof Wellbore
    */
-  name: string;
+  externalId?: string;
   /**
-   * Custom, application specific metadata. String key -> String value.
-   * @type {{ [key: string]: string; }}
+   * Parent asset id in CDF
```

Comment:
The well is not necessarily the parent of the wellbore, it could be higher up above the hierarchy. This doc string should probably state something more like "CDF id of the well that contains this wellbore"